src/main/app/controller/other-names/OtherNamesPostController.ts

```diff
@@ -47,6 +48,9 @@ export default class OtherNamesPostController extends PostController<AnyObject>
           },
           this.getEventName(req)
         );
+      } else if (formData[`${this.fieldPrefix}OtherFirstNames`] && formData[`${this.fieldPrefix}OtherLastNames`]) {
+        req.session.errors.push({ propertyName: 'addButton', errorType: ValidationError.ADD_BUTTON_NOT_CLICKED });
```

Comment:
There is an accessibility issue with this logic, when the error appears, it is linked to addButton in the errorSummary but when we click the error message, nothing happens
Can you instead link this error to top level question `${this.fieldPrefix}HasOtherNames` ?